bin/resin-preload

```diff
@@ -2,9 +2,12 @@
 
 'use strict'
 
-const fs = require('fs')
+const Docker = require('dockerode')
 const preload = require('..')
 const info = require('../package.json')
+const streamToPromise = require('stream-to-promise')
+
+const docker = new Docker({Promise})
```

Comment:
Yeah I figured as much, and guessed dockerode would default to the local docker socket.